src/views/html-imports.vue

```diff
@@ -82,9 +92,13 @@ export default {
   },
   methods: {
     toggleShowDupes() {
-      if(!this.showDuplicates && currentImports.length > 100) {
-        const result = confirm(`You seem to have a lot of imports (${currentImports.length}), showing duplicates can slow things up, still do it?`);
-        if(!result) return;
+      if (!this.showDuplicates && currentImports.length > 100) {
```

Comment:
Please remove the logic for showDuplicates if we are not using it.

Reply:
Done.